### DISCUSSION TITLE
Add support for bundle articles

### DISCUSSION THREAD

**Discussion by nicklasv** [FortnoxAB/csharp-api-sdk#312] [General]
Fortnox has released 4 new endpoints, handling the bundle articles.
Can I contribute to add these new endpoints in the source?

https://www.fortnox.se/developer/blog/new-feature-bundle-articles

**Comment by adamelfstrom** [comment]
Hi, a couple of weeks ago issue #309 was created for this feature, but we have not started to implement it in the SDK yet.
You are welcome to contribute by creating a PR 🙂
